### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by hklages** [Foddy/node-red-contrib-huemagic#180]
**Describe the bug**
Since the last update to 2.8.1 almost every second hour I receive the above message.  It is one out of 3 group nodes causing the error. It shows also the status "input error". All other nodes work fine so far.  I use catch node for error handling.

**Flow to Reproduce**
```
[{"id":"16ae5e7c.1dbc02","type":"change","z":"b06f8da5.377a8","name":"false","rules":[{"t":"set","p":"payload","pt":"msg","to":"false","tot":"bool"}],"action":"","property":"","from":"","to":"","reg":false,"x":709.5,"y":2658,"wires":[["e78cdd20.dc88d","6747e3e4.4c3f9c"]]},{"id":"6747e3e4.4c3f9c","type":"hue-group","z":"b06f8da5.377a8","name":"All","bridge":"106cd2a8.cd21cd","groupid":"10","colornamer":true,"skipevents":false,"x":950,"y":2580,"wires":[[]]},{"id":"106cd2a8.cd21cd","type":"hue-bridge","z":"","name":"Philips hue","bridge":"192.168.178.26","key":"ZSzjbE2z5j2dr3NjKt8E8JuSHMt51eP7FCROJWrY","interval":"3000"}]
```

**Expected behavior**
No error message.

**Please complete the following information:**
 - Node-RED version: 1.0.5
 - NodeJS version: 10.20.0
 - Device running Node-RED: [CCU3 with RedMatic]
- HUEMAGIC: 2.8.1

**Comment by GeorgeCastanza** [comment]
I am also having issues with **group** HueMagic nodes.  

Hitting deploy will often (but not always) cause errors, whereas, the same flow with an inject node beginning the flow will not. It's not reliably reproducible.  I nearly posted something yesterday, but dont have a flow where occurs reliably.

**Comment by hklages** [comment]
solved with release 2.8.2
Thanks!

**Comment by GeorgeCastanza** [comment]
Yes, agreed. My issue is solved. Thanks
